### ISSUE TITLE
Navigation: Collapsible Menu Items Labelled Incorrectly

### ISSUE THREAD

**Issue by donalirl** [Automattic/wp-calypso#41706]
#### Steps to reproduce

1. Starting at WordPress.com while logged in
2. User a screen reader to navigate through the options under My Site that are shown here https://d.pr/i/xISLUc
3. Listen to what the screen reader calls the collapsible menu items (Site, Manage, Tools, etc.)

#### What I expected

I expected the menu items to clearly communicate what they were and how to interact with them.

#### What happened instead

‘Manage’ reads as something like “three group image manage image” with Apple Voiceover.

#### Comments

@enriquesanchez helped me reproduce this and says:

> I believe this is happening because the icons (both the menu item action plus the ‘arrow down’) are being read aloud by VoiceOver, that’s why you get a “Heading level 2, 3 items: image, Design, image).
> 
> [I think this can be avoided if we add an aria-hidden to the icons. ](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute)I don’t think they provide any meaningful content to a screen reader, and as such should be hidden.
> 
> Another thing that caught my eye is why these menu items are marked up as heading level 2, while the other ones aren’t. I don’t think they need to be h2‘s, since they are navigation items.

#### Browser / OS version

Safari with Voiceover

#### Screenshot / Video

![2020-04-28-15-30-00 2020-04-28-15_32_03](https://user-images.githubusercontent.com/30727666/80781949-9f883000-8bb7-11ea-87a3-1ee770c6a093.gif)

#### Context / Source

#p4vt7e-6n-p2

#journey #empathy #livesharing #accessibility #painpoint #user-report

**Comment by donalirl** [comment]
In addition, these menu items should be read by a screen reader as "expanded" when expanded and "collapsed" when collapsed. This is important for users who rely on screen readers.

**Comment by richardkang112** [comment]
Hi. May I work on this issue? @donalirl

**Comment by donalirl** [comment]
All good with me @richardkang112

**Comment by richardkang112** [comment]
@donalirl Could you please check out my branch [here](https://github.com/richardkang112/wp-calypso/tree/fix/collapsible-menu-items-labelled-incorrectly) and find out how it's read by apple voiceover if you don't mind?  The following is the before and after for Chrome Screen Reader:

Before:
"List item lists one items. Posts. Heading 2 collapsed list item."

After:
"List item lists one items. Posts. Link collapsed list item."

When the menu is expanded, the screen reader reads "link expanded list item." Instead of "link collapsed list item."

**Comment by donalirl** [comment]
Hi @richardkang112 I'm not able to do that, sorry. Would you be able to ask someone else?

**Comment by richardkang112** [comment]
@donalirl No problem. I'll try to find a way to test my fix on Safari with Voiceover.

**Comment by obenland** [comment]
Is this bug still something that occurs after Nav Unification?

/cc @tjcafferkey

**Comment by sixhours** [comment]
I can confirm that the icons are no longer read by Voiceover after nav unification.

> Another thing that caught my eye is why these menu items are marked up as heading level 2, while the other ones aren’t. I don’t think they need to be h2‘s, since they are navigation items.

This is still happening; every item is read as heading level two, and there's no indication that menu items can be expanded/collapsed.

**Comment by sixhours** [comment]
I tested @richardkang112's branch and confirmed that changing the `h2`s to anchors both simplifies what's read by Voiceover when you navigate the menu, and it adds the expanded/collapsed state:

<img width="797" alt="Screen Shot 2021-08-05 at 3 24 35 PM" src="https://user-images.githubusercontent.com/2124984/128409252-c6290232-2b60-4507-b7a0-1b19a6d7e380.png">

This seems like a big accessibility win and doesn't seem to negatively affect the navigation otherwise. @richardkang112 would you be up for creating a new PR to this repository with your changes so we can push it through? The icon change is no longer necessary but the links change would be great.

**Comment by richardkang112** [comment]
Hi @sixhours, thank you for testing the branch. I'll be happy to create the PR for this issue.
